docs/guide/configuration/mqtt.md

```diff
@@ -12,6 +12,7 @@ Zigbee2MQTT requires a MQTT-Server connection to operate.
 # Required: MQTT settings
 mqtt:
     # Required: MQTT server URL (use mqtts:// for SSL/TLS connection)
+    # Note: Use hostname 'core-mosquitto' or 'homeassistant' when using Z2M as an addon inside Home Assistant and when using Mosquito Home assistant addon as MQTT broker
```

Comment:
```suggestion
    # Example: 'localhost', when using the Mosquito HA addon use 'core-mosquitto'
```

I'm not sure about the `homeassistant` one, when did you see it?

Reply:
I have checked on my side, to be sure. It works with `core-mosquito` or `homeassistant` but not with `localhost`. I think we can keep only `core-mosquito`.